bundles/org.openhab.binding.surepetcare/src/main/java/org/openhab/binding/surepetcare/internal/dto/SurePetcarePetActivity.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.surepetcare.internal.dto;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Date;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link SurePetcarePetActivity} is the Java class used to represent the
+ * status of a pet. It's used to deserialize JSON API results.
+ *
+ * @author Rene Scherer - Initial contribution
+ * @author Holger Eisold - Added pet feeder status
+ */
+public class SurePetcarePetActivity {
+
+    @SerializedName("tag_id")
```

Comment:
can you explain what you mean and where do i find/edit this setting? Thanks

Reply:
https://stackoverflow.com/a/11810183/3656774

I expect some other openHAB bindings will also use it.

Reply:
thanks...
okay i got/understood this... and we still use this FieldNamingPolicy in the binding... I removed all the unneeded `@SerializedName` where possible...

Some are atill present... since we changed the fields in the binding....
Is there a negative impact to use this `@SerializedName` annotation? or can we leave them for now?